columnflow/tasks/yields.py

```diff
@@ -0,0 +1,191 @@
+                h_in = inp["event"].load(formatter="pickle")
+
+                # loop and extract one histogram per process
+                for process_inst in process_insts:
+                    # skip when the dataset is already known to not contain any sub process
+                    if not any(map(dataset_inst.has_process, sub_process_insts[process_inst])):
+                        continue
+
+                    # work on a copy
+                    h = h_in.copy()
+
+                    # axis selections
+                    h = h[{
+                        "process": [
+                            hist.loc(p.id)
+                            for p in sub_process_insts[process_inst]
+                            if p.id in h.axes["process"]
+                        ],
+                    }]
+
+                    # axis reductions
+                    h = h[{"process": sum, "shift": sum, "event": sum}]
+
+                    # add the histogram
+                    if process_inst in hists:
+                        hists[process_inst] += h
+                    else:
+                        hists[process_inst] = h
+
+            # there should be hists to plot
+            if not hists:
+                raise Exception("no histograms found to plot")
+
+            # sort hists by process order
+            hists = OrderedDict(
+                (process_inst, hists[process_inst])
+                for process_inst in sorted(hists, key=process_insts.index)
+            )
+
+            yields = []
+            yield_header = ["Process"] + [category_inst.label for category_inst in category_insts]
+
+            for process_inst, h in hists.items():
+                row = []
+                row.append(process_inst.label)
+
+                for category_inst in category_insts:
+                    leaf_category_insts = category_inst.get_leaf_categories() or [category_inst]
+
+                    h_cat = h[{"category": [
+                        hist.loc(c.id)
+                        for c in leaf_category_insts
+                        if c.id in h.axes["category"]
+                    ]}]
+                    h_cat = h_cat[{"category": sum}]
+
+                    value = Number(h_cat.value, math.sqrt(h_cat.variance))
+
+                    # TODO: allow normalizing per process or per category (or both?)
+                    if self.skip_uncert:
+                        value_str = value.str(format=self.number_format).split(" +- ")[0]
+                    else:
+                        value_str = value.str(
+                            format=self.number_format,
+                            style="latex" if "latex" in self.table_format else "plain",
+                        )
+
+                    row.append(value_str)
+
+                yields.append(row)
+
+            yield_table = tabulate(yields, headers=yield_header, tablefmt=self.table_format)
+            print(yield_table)
```

Comment:
`self.publish_message(yield_table)`

-> this prints the table, but also forwards it to any connected central scheduler so that one can see the table in the web interface.